autoware_launch/launch/autoware.launch.xml

```diff
@@ -7,13 +7,31 @@
   <arg name="use_pointcloud_container" default="true" description="launch pointcloud container"/>
   <arg name="pointcloud_container_name" default="pointcloud_container"/>
   <!-- Optional parameters -->
+  <arg name="vehicle_id" default="$(env VEHICLE_ID default)" description="vehicle specific ID"/>
   <arg name="rviz" default="true" description="launch rviz"/>
+  <arg name="rviz_config" default="$(find-pkg-share autoware_launch)/rviz/autoware.rviz" description="rviz config"/>
   <arg name="lanelet2_map_file" default="lanelet2_map.osm" description="lanelet2 map file name"/>
   <arg name="pointcloud_map_file" default="pointcloud_map.pcd" description="pointcloud map file name"/>
+  <arg name="launch_vehicle" default="true" description="launch vehicle"/>
+  <arg name="launch_vehicle_interface" default="true" description="launch vehicle interface"/>
+  <arg name="launch_system" default="true" description="launch system"/>
+  <arg name="system_run_mode" default="online" description="run mode in system"/>
+  <arg name="launch_map" default="true" description="launch map"/>
+  <arg name="launch_sensing" default="true" description="launch sensing"/>
+  <arg name="launch_sensing_driver" default="true" description="launch sensing driver"/>
+  <arg name="launch_localization" default="true" description="launch localization"/>
+  <arg name="launch_perception" default="true" description="launch perception"/>
+  <arg name="launch_planning" default="true" description="launch planning"/>
+  <arg name="launch_control" default="true" description="launch control"/>
+  <arg name="init_simulator_pose" default="false" description="initialize simulator pose"/>
+  <arg name="init_localization_pose" default="true" description="initialize localization pose"/>
```

Comment:
This parameter can be merged into `launch_localization`.

Reply:
@isamu-takagi 
What do you mean by "can be merged".
init_localization_pose flag is opposite to launch_localization flag?

Reply:
@takayuki5168 
`init_localization_pose` should be true when the localization module is launched. So this parameter has the same value as `launch_localization`.

Reply:
https://github.com/autowarefoundation/autoware_launch/pull/59/commits/e4b27733944a710ab9de4569639776ea148809c3
DONE